linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_deny_root/rule.yml

```
@@ -21,6 +21,8 @@ identifiers:
 
 references:
     anssi: BP28(R18)
+    cis@sle12: 5.3.2
+    cis@sle15: 5.3.2    
     cis-csc: 1,12,15,16
     cobit5: DSS05.04,DSS05.10,DSS06.10
```

`accounts_passwords_pam_tally2_deny_root/rule.yml has one or more unsorted references`

They should be after the cis-csc reference. Similar as in: https://github.com/ComplianceAsCode/content/blob/328eac5d78ee756d158c389a91633f5dd74a5d60/linux_os/guide/system/software/integrity/software-integrity/aide/aide_use_fips_hashes/rule.yml#L26-L29

```suggestion
references:
    anssi: BP28(R18)
    cis-csc: 1,12,15,16
    cis@sle12: 5.3.2
    cis@sle15: 5.3.2
    cobit5: DSS05.04,DSS05.10,DSS06.10
```